app/controllers/api/v1/admin/project.js

```diff
@@ -2,6 +2,12 @@ const { ProjectStatus } = global.appEnums;
 const moment = require('moment');
 const { sumBy, flatMap, map, filter } = require("lodash");
 const { asyncMiddleware } = global;
+const {
+  sumBy,
+  flatMap,
+  map,
+  filter
```

Comment:
Where are we using these methods?

Reply:
In some APIs, we are using these methods.